select/sdks/android/guide-v1.md

```diff
@@ -60,8 +60,10 @@ implementation 'com.github.FidelLimited:android-sdk:1.7.5'
 - Go to the `Plan` tab and copy your `Test` or `Live` SDK Key.
 - Set your SDK Key in your app:
 
+> Important note: For security reasons, please do not store the SDK Key in your codebase. Follow our [SDK security guide](/docs/stream/sdks/sdk-security-guidelines) for detailed recommendations.
```

Comment:
Should we always add emphasis? "**DO NOT**" instead of just "do not"?

Reply:
I see only benefits to doing this, so yes, I’ll emphasize 😃.